lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -30,20 +31,24 @@
 
 <li class="course-item">
   % if settings.FEATURES.get('ENABLE_VERIFIED_CERTIFICATES'):
-    % if enrollment.mode == "verified":
-      % if verification_status.get('status') in [VERIFY_STATUS_NEED_TO_VERIFY, VERIFY_STATUS_SUBMITTED, VERIFY_STATUS_APPROVED]:
-        <% mode_class = " verified" %>
-      % else:
-        <% mode_class = " honor" %>
-      % endif
-    % else:
-      <% mode_class = " " + enrollment.mode %>
-    % endif
+    <% course_verified_certs = CourseMode.get_certificate_display(enrollment.mode, verification_status.get('status')) %>
```

Comment:
In case of empty enrolment it will show the ' ' in html. This method will return default values.  show_image = False
enrollment_title, enrollment_value, image_alt = '', '', ''